libraries/AP_HAL_ChibiOS/hwdef/KakuteH7v2/defaults.parm

```diff
@@ -3,4 +3,9 @@ SERVO9_FUNCTION 120
 NTF_LED_TYPES 257
 # Default VTX power to on
 RELAY_DEFAULT 1
-SERIAL7_PROTOCOL 16
\ No newline at end of file
+# UART1 for DJI Goggles
+SERIAL1_PROTOCOL 33
+# UART3 for VTX
+SERIAL3_PROTOCOL 37
+#ESC telem on RX7
+SERIAL7_PROTOCOL 16
```

Comment:
This is actually unnecessary - already defined in hwdef.dat

Reply:
did not add that...but can remove...

Reply:
Yeah, realised - just seemed worth fixing while you are in here